packages/components/ovh-shell/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@ovh-ux/manager-config": "^5.0.0 || ^6.0.0",
```

Comment:
**issue(blocking):** `@ovh-ux/shell` cannot be published to the NPM registry because `@ovh-ux/manager-config` is a private package.

Reply:
resolved in main branch